module/batch-action/src/Domain/Repository/BatchActionRepositoryInterface.php

```diff
@@ -11,12 +11,17 @@
 use Ergonode\BatchAction\Domain\Entity\BatchActionId;
 use Ergonode\BatchAction\Domain\Entity\BatchAction;
 use Ergonode\SharedKernel\Domain\AbstractId;
+use Ergonode\SharedKernel\Domain\AggregateId;
 
 interface BatchActionRepositoryInterface
 {
     public function load(BatchActionId $id): ?BatchAction;
 
     public function save(BatchAction $batchAction): void;
 
-    public function addEntry(BatchActionId $id, AbstractId $entryId): void;
+    public function addResource(BatchActionId $id, AbstractId $entryId): void;
```

Comment:
When you `addResource` use `AbstractId` but when we mark you use `AggregateId` maybe change to one master type

Reply:
It should be `addEntry`